app/components/widgets/AlertBox.vue

```diff
@@ -89,7 +92,7 @@
       <v-form-group v-model="selectedVariation.settings.text.color2" :metadata="metadata.secondaryColor" />
     </div>
   </validated-form>
-  <validated-form slot="media-properties" key="media-properties" @input="save()" v-if="selectedVariation">
+  <validated-form slot="media-properties" key="media-properties" @input="handleFacemaskInput()" v-if="selectedVariation">
```

Comment:
Nitpick: remove brackets.

Reply:
again not sure about which brackets you're referring to... i dont even see any here